augur/data/__init__.py

```diff
@@ -29,4 +29,7 @@
 
 
 def as_file(path: str) -> ContextManager[Path]:
-    return _as_file(_files(__name__) / path)
+    return _as_file(files(path))
+
+def files(path: str):
+    return _files(__name__) / path
```

Comment:
This commit should also update the header at the top of the file which says "Copied without modification" even though the commit modifies it. (However, modification is not strictly necessary if you'd rather avoid it… see below.)

Reply:
Updated to not modify, but also changed the wording to be more future-proof:

> Originally copied from nextstrain/cli/resources/__init__.py in the Nextstrain CLI project¹.